.circleci/config.yml

```diff
@@ -36,6 +36,36 @@ jobs:
           paths:
             - ui/node_modules
 
+  go-mod-vendor:
+    docker:
+      - image: *GOLANG_IMAGE
+    working_directory: /go/src/github.com/hashicorp/vault
+    steps:
+      - add_ssh_keys:
```

Comment:
I don't think we should need this for OSS, right?

Reply:
correct, good catch!  dropped.